### DISCUSSION TITLE
GoLang Coders Wellington NZ

### DISCUSSION THREAD

**Discussion by MarkAngus** [community/community#27989] [Other Feature Feedback, Questions, & Ideas]
Hi 
I am looking for a couple of NZ based GoLang Coders (ideally in Wellington), kif there is anyone here who is interested in getting involved in a FinTech Project, which involves retrieval of a Datafeed, parsing of this, and capture in to a cloud database, where an API is attached.  There is also a React JS (SPA) coding requirement for the front end, although my primary interest is in finding GoLang coders...
Thanks
Mark

**Comment by MarkAngus** [comment]
Thanks - have filled the roles since posting this, however will keep in mind :-)

**Comment by otter-computer** [comment]
Hi @MarkAngus, we're happy you're here 😀

However, we currently do not allow job listings on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
